rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -741,31 +755,68 @@ public String getLockWhitelistAddress(Object[] args)
         logger.trace("getLockWhitelistAddress");
 
         int index = ((BigInteger) args[0]).intValue();
-        String address = bridgeSupport.getLockWhitelistAddress(index);
+        LockWhitelistEntry entry = bridgeSupport.getLockWhitelistEntryByIndex(index);
 
-        if (address == null) {
+        if (entry == null) {
             // Empty string is returned when address is not found
             return "";
         }
 
-        return address;
+        return entry.address().toBase58();
+    }
+
+    public long getLockWhitelistEntryByAddress(Object[] args)
+    {
+        logger.trace("getLockWhitelistEntryByAddress");
+
+        String addressBase58;
+        try {
+            addressBase58 = (String) args[0];
+        } catch (Exception e) {
+            logger.warn("Exception in getLockWhitelistEntryByAddress: {}", e);
+            return -1;
```

Comment:
I'd assign a different return value for an invalid formatted address (-2 perhaps?)